doc/deployment_guide.md

```diff
@@ -0,0 +1,6080 @@
+a.	Creation of new bonds for the new ports that will be used for OVS-DPDK bond with LACP enabled on two ports.
+b.	Removal of Removal of tenant VLANs from NOVA1-bond0, NOVA2-bond0 and NOVA3-bond0 bonds.
+c.	Tagging of newly created bonds in tenant network VLANs.
+
+![Figure 3: OVS-DPDK with two ports reference wiring diagram](media/88677057ea84a8e286c164657821eb14.jpg)
+
+> NOTE: In the reference diagram above, PCI Slot 1 and PCI Slot 2 are utilized on all the Compute nodes. PCI slots on the Compute nodes are connected to swp38, swp40, and swp42 interfaces on Leaf-1 and Leaf-2 switches according to the reference diagram.
+
+
+Switch configurations for OVS-DPDK on both switches according to the four ports OVS-DPDK will be:
+
+```bash
+$ net add interface swp38,swp40,swp42
+$ net add bond NOVA1-bond10 bond slaves swp38 clag id 59
+$ net add bond NOVA2-bond10 bond slaves swp40 clag id 61
+$ net add bond NOVA3-bond10 bond slaves swp42 clag id 63
+$ net add bond NOVA1-bond10 bridge vids 201-220
+$ net add bond NOVA2-bond10 bridge vids 201-220
+$ net add bond NOVA3-bond10 bridge vids 201-220
+
+
+Also, the port channel will have the following configuration on both switches:
+
+auto NOVA1-bond10
+iface NOVA1-bond10
+   bond-slaves swp38
+   bridge-vids 201-220
+   clag-id 59
+   mtu 9000
+ 
+ 
+auto NOVA2-bond10
+iface NOVA2-bond10
+   bond-slaves swp40
+   bridge-vids 201-220
+   clag-id 61
+   mtu 9000
+ 
+ 
+auto NOVA3-bond10
+iface NOVA3-bond10
+   bond-slaves swp42
+   bridge-vids 201-220
+   clag-id 63
+   mtu 9000
+```
+
+
+
+**OVS-DPDK with Four Ports:**
+
+
+For this scenario, switch configuration will have three major changes:
+
+a.	Creation of new bonds for the new ports that will be used for OVS-DPDK bond with LACP enabled on four ports.
+
+b.	Removal of tenant VLANs from NOVA1-bond0, NOVA2-bond0 and NOVA3-bond0 bonds.
+c.	Tagging of newly created bonds in tenant network VLANs.
+
+    > Note: Tenant VLANs in JS 13.0:
+    > Mention the VLAN range for tenant network from 201-220 in the INI file under SAH node.i.e. Default Tenant VLANs range: 201-220
+
+![Figure 4: OVS-DPDK with four ports reference wiring diagram](media/356efb0038cfa4229f3989a8ad873c02.jpg)
+
+> Note: In the four ports OVS-DPDK reference diagram, PCI Slot 1 and PCI Slot 2 are utilized on all the Compute nodes. PCI slots on the Compute nodes are connected to swp38, swp39, swp40, swp41, swp42, and swp43 interfaces on Leaf-1 and Leaf-2 switches according to the reference diagram.
+
+**Switch configurations for OVS-DPDK on both switches according to the four ports OVS-DPDK will be:**
+
+
+```bash
+$ net add interface swp38-43
+$ net add bond NOVA1-bond10 bond slaves swp38-39 clag id 59
+$ net add bond NOVA2-bond10 bond slaves swp40-41 clag id 61
+$ net add bond NOVA3-bond10 bond slaves swp42-43 clag id 63
+$ net add bond NOVA1-bond10 bridge vids 201-220
+$ net add bond NOVA2-bond10 bridge vids 201-220
+$ net 
```

Comment:
Update above to new solution name "Dell EMC Ready Architecture for Red Hat OpenStack Platform v13"